src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -39,7 +39,7 @@
      * Root node of any Javadoc comment.
      * Last child is always {@link #EOF}.
      *
-     * <p>
+     * <p/>
```

Comment:
Why do we have an empty `p` tag? If this is to enforce new lines in html/report, shouldn't we use `<br />` or surround whole text in `<p>`.

In this case, I think we should surround `<b>Tree for example:</b>` with `<p>`.
=> `<p><b>Tree for example:</b></p>`

Reply:
[Here](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format) it is said that we should separate paragraphs with `<p>` tag. But if we don't care on this old document, I would rather rewrite it as you have said: `<p><b>Tree for example:</b></p>`, as `<br/>` tag is also old-fashioned solution.

Reply:
@romani Can confirm, but in xdocs we write paragraphs surrounding the entire text. I feel we should follow suite in the Javadocs.
Ex: https://github.com/checkstyle/checkstyle/blob/fd7fb45d409a2de0dead15f907e69b330dbaa5dc/src/xdocs/config_blocks.xml#L26-L38

> Here it is said that we should separate paragraphs with <p> tag.

To me this also looks like the old way of writing HTML before XHTML came and standardized the layout. A new PR will also prevent some Javadoc checks from executing on unstructured Javadocs so this is another reason why should avoid this way of writing.

Reply:
fixed

Reply:
confirmed.
format: `<p>...</p>` is expected. in future this Check will work on XHTML-like format, and all of our javadocs will become XHTML-like. Event it is not favorite form for HTML.